@utahdts/utah-design-system/react/components/containers/tabs/TabGroup.jsx

```diff
@@ -38,12 +50,39 @@ export function TabGroup({
           setTabGroupState((draftState) => { draftState.selectedTabId = tabId; });
         }
       },
+      navigateNext: () => {
+        const index = findCurrentTabIndex();
+        if (index !== (tabGroupState.tabs.length - 1)) {
+          navigateTab(tabGroupState?.tabs?.[index + 1] || null);
+        } else {
+          navigateTab(tabGroupState?.tabs?.[0] || null);
+        }
+      },
+      navigatePrevious: () => {
+        const index = findCurrentTabIndex();
+        if (index !== 0) {
+          navigateTab(tabGroupState?.tabs?.[index - 1] || null);
+        } else {
+          navigateTab(tabGroupState?.tabs?.[tabGroupState.tabs.length - 1] || null);
+        }
+      },
+      isVertical: !!isVertical,
     }),
     [onChange, setTabGroupState, tabGroupState, value]
```

Comment:
do all these need to be in the dependency list since they don't seem to be all used in this function? and shouldn't navigateTab be in this list?